### ISSUE TITLE
Slow server start with a lot of files

### ISSUE THREAD

**Issue by MauriceNino** [mtoensing/Docker-Minecraft-PaperMC-Server#75]
I have `Dynmap` installed on my server and preloaded a few thousands of chunks with `Chunky`, which results in a lot of files for the map. Starting up the server now takes close to a minute, because my log gets spammed with the following.

```
ownership of '/data/plugins/dynmap/web/tiles/world/t/x_x/x_x.jpg' retained as dockeruser:dockergroup
```

Is there any need to change the ownership? Wouldn't it be better to just assume correct permission handling? At least that's how I know it from other services.

Alternatively, an environment variable like `-e SKIP_PERM_CHECK="true"` would be handy.

**Comment by mtoensing** [comment]
Yes, good idea. Are you able to provide a pull request m?

**Comment by MauriceNino** [comment]
Sure, I can definitely provide a PR. Which solution would you prefer? Do you think that the `chown` is actually needed, or would you like to remove it?
Or would you prefer the env var solution?

**Comment by mtoensing** [comment]
To be honest I am not sure why Someone added this ownership process at every start but it solved a problem back then. That was a PR, too.

i would say an environment variable should be great to disable it. What do you say?

**Comment by MauriceNino** [comment]
Created a PR for it. Feel free to make any necessary changes!

**Comment by mtoensing** [comment]
Solved thanks to #76
